modules/acquia_cms_common/acquia_cms_common.info.yml

```diff
@@ -5,7 +5,6 @@ type: module
 core_version_requirement: ^9
 dependencies:
   - drupal:acquia_purge
-  - drupal:acquia_telemetry
```

Comment:
Is it on purpose to keep the module in composer.json of aquia_cms_common

<img width="491" alt="Screenshot 2022-10-20 at 7 04 08 PM" src="https://user-images.githubusercontent.com/106175376/196963083-6abba61a-9493-46e3-9a7c-9e60fae4b24a.png">

Reply:
@apathak18 - If it is removed from composer then existing sites breaks as acquia_telemetry is installed and missing.